src/test/java/de/adesso/kicker/notification/Message/SendMessageServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package de.adesso.kicker.notification.Message;
+
+import de.adesso.kicker.user.UserDummy;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class SendMessageServiceTest {
+
+    @Mock
+    private MessageRepository messageRepository;
+
+    @InjectMocks
+    private SendMessageService sendMessageService;
+
+    @BeforeAll
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    @DisplayName("Save a decline Message")
+    void saveDeclineMessage() {
+        // given
```

Comment:
Given status is empty

Reply:
no given needed in this primitive function

Reply:
In given status you should generate all objects that are needed, in the following you are using 2x `UserDummys`. It's easier to read if you do it like suggested and please name them descriptive its not clear, what `sendMessage` does. Are these 2 senders, 2 receivers or whatever? This was only a rhetorical question.